src/Avalonia.Visuals/Media/Typeface.cs

```diff
@@ -7,18 +7,8 @@ namespace Avalonia.Media
     /// </summary>
     public class Typeface
     {
-        /// <summary>
-        /// Initializes a new instance of the <see cref="Typeface"/> class.
-        /// </summary>
-        /// <param name="fontFamilyName">The name of the font family.</param>
-        /// <param name="fontSize">The font size, in DIPs.</param>
-        /// <param name="style">The font style.</param>
-        /// <param name="weight">The font weight.</param>
-        public Typeface(
-            string fontFamilyName,
-            double fontSize,
-            FontStyle style = FontStyle.Normal,
-            FontWeight weight = FontWeight.Normal)
+        public Typeface(FontFamily fontFamily, double fontSize = 12, FontStyle style = FontStyle.Normal,
```

Comment:
Need to add documentation to this method.

Reply:
I think I should spend some time on comments in general. There is a lot room for improvements.